apps/src/clientApi.js

```diff
@@ -49,12 +50,7 @@ require('./utils');
  *        method being invoked.
  */
 
-/** 
- * Namespace for the client API for accessing channels, tables and properties.
- */
-var clientApi = module.exports;
-
-var ApiRequestHelper = function (baseUrl) {
+var ClientApiRequest = function (baseUrl) {
```

Comment:
Tiny nit: I never had JSDoc on `ApiRequestHelper` here (I should have) but removal of the doc'd `var clientApi = ...` line has caused my linter to incorrectly associate the JSDoc callback definition above with this line.  Might be nice to doc this.
```
/**
 * Base class for API request helpers.
 * @param {string} baseUrl
 * @constructor
 */
var ClientApiRequest = function (baseUrl) {
```